src/lib/Interface.js

```diff
@@ -372,7 +372,7 @@ export default class Interface {
   }
   
   setCookie(cookie, callback) {
-    const expires_in = new Date(Date.now() + 365 * 24 * 60 * 60 * 1000).toUTCString();
+    const expires_in = new Date(Date.now() + 30 * 24 * 60 * 60 * 1000).toUTCString();
```

Comment:
code is fine title is maybe not as 30 days != month in many cases and not sure if we need to account for things like 28 and 31 day months.

Reply:
Thomas confirmed that it should always be 30 days. Good point raising this question!